tests/e2e/app/src/App.js

```diff
@@ -0,0 +1,40 @@
+import React, {useMemo} from 'react';
+import {Text, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import Main from './Main';
+import ONYXKEYS from './keys';
+
+const propTypes = {
+    session: PropTypes.shape({login: PropTypes.string}),
+};
+
+const defaultProps = {
+    session: {},
+};
+
+function App(props) {
+    const isAuthenticated = useMemo(
+        () => Boolean(lodashGet(props.session, 'login', null)),
+        [props.session],
+    );
```

Comment:
`useMemo` looks like unnecessary here. 
```suggestion
    const isAuthenticated = Boolean(lodashGet(props.session, 'login', null));
```